### ISSUE TITLE
Remove the spellchecker in commit messages - it is usually incorrect

### ISSUE THREAD

**Issue by adamplural** [desktop/desktop#13413]
### Describe the bug

When typing a commit message, this app attempts to do a spellcheck. Since commit messages often contain variable-names, project names, etc, this spellcheck is nearly ALWAYS incorrect.

Spellcheck should be 'opt-in' for the people who have spelling/language issues and desire that prompt. By default it should not be interfering with correct messages and claiming they are errors.

### Version & OS

2.9.4 / Windows-10

### Steps to reproduce the behavior

1. Cause a change to a file
2. In Commit-message field, type a variable name from your project in the commit message
3. GHD incorrectly underlines the var name in red

### Expected behavior

Nothing

### Actual behavior

False 'error' report

**Comment by tidy-dev** [comment]
@adamplural Sorry for your frustrations. However, there is an easy way to disable it. Right click the commit message or description and select "Disable Commit Spellcheck".

**Comment by adamplural** [comment]
It's nice to have that as a convenience, but this is very non-standard -
all other apps put this as an option in their Options/Preferences menu.
I've never seen a core app feature like this hidden in the right-click
menu. Surely it should be at least in both places?

**Comment by tidy-dev** [comment]
We did discuss adding it there, but we strive for keeping preferences to very broad high impact settings. Also, seeing as it is common to interact with spellcheckers by right clicking on a misspelled word to open a context menu (to find a correction), we decided that this was a more convenient placement and would give users who do not like spellcheckers a very quick way to turn it off. :) Thank you for your feedback!
